pulp_ansible/app/constants.py

```diff
@@ -0,0 +1,2 @@
+# default results per page. used to calculate number of pages
+PAGE_SIZE = 10
```

Comment:
What if we use a bigger number like 100 to reduce the number of page requests?

Reply:
sounds good to me, I'll update it